clippy_lints/src/missing_doc.rs

```diff
@@ -35,6 +36,8 @@ declare_clippy_lint! {
 }
 
 pub struct MissingDoc {
+    /// Whether to only check for missing docs in `pub(crate)` items.
```

Comment:
Can this be updated in the same way as https://github.com/rust-lang/rust-clippy/pull/10303/files#diff-1d884fd439e15a9e47e1135cdc0ab5ee896a876b9f4c256ecce4be6c69a8831dR459?